components/content/inspira/ui/flip-card/FlipCard.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div :class="cn('group h-72 w-56 [perspective:1000px]', props.class)">
+    <div
+      :class="
+        cn(
+          'relative h-full rounded-2xl transition-all duration-500 [transform-style:preserve-3d]',
+          rotation[0],
+        )
+      "
+    >
+      <!-- Front -->
+      <div class="absolute size-full [backface-visibility:hidden]">
+        <NuxtImg
```

Comment:
Please update the component to use  `<img>`.
The components must support Vue and Nuxt, and `NuxtImg` is Nuxt specific.

Reply:
Done, but @rahul-vashishtha I just checked in APP, we have used NuxtImg in many components, do we need to update in all the places?

Reply:
Yes, those components need to be updated too. I'll work on that once this PR is merged